src/explore-education-statistics-common/src/components/__tests__/Accordion.test.tsx

```diff
@@ -314,4 +314,19 @@ describe('Accordion', () => {
 
     expect(toggleAll).toHaveBeenCalledWith(false);
   });
+
+  test('it renders visuallyHiddenText', () => {
+    const { container } = render(
+      <Accordion
+        id="test-sections"
+        visuallyHiddenText="Academic Year 2016/17 sections"
+      >
+        <AccordionSection heading="Test heading">Test content</AccordionSection>
+      </Accordion>,
+    );
+
+    expect(container.querySelectorAll('.govuk-visually-hidden')).toHaveLength(
+      1,
+    );
```

Comment:
Can you test that it renders the text here rather than just that there's a div with the className?